### PR TITLE
feat: add support to set all the arguments of greptime helm chart

### PR DESCRIPTION
**Pull request by shawnh2**
related to #91

### CODE REVIEW

File: pkg/cmd/gtctl/cluster/create/create.go [hunk 1]
```diff
@@ -57,8 +57,11 @@ type createClusterCliOptions struct {
 	AlwaysDownload     bool
 
 	// Common options.
-	Timeout int
-	DryRun  bool
+	Timeout     int
+	DryRun      bool
+	Set         string
```
Comment (zyy17) [inline pkg/cmd/gtctl/cluster/create/create.go]: suggestion: maybe we just need one `Set`(**to make cli more clean**) and use the some format to differentiate different components? For example:

```
gtctl cluster create mycluster \
--set cluster.frontend.replicas=1 \
--set etcd.storageClassName=standard \
--set operator.replicas=2
```

For the above example, I use `cluster`/`etcd`/`operator` as the identifier of the first part of the string.

File: pkg/helm/helm.go [hunk 2]
```diff
@@ -98,8 +98,12 @@ func (r *Render) GenerateHelmValues(input interface{}) (map[string]interface{},
 	typeOf := reflect.TypeOf(input)
 	for i := 0; i < valueOf.NumField(); i++ {
 		helmValueKey := typeOf.Field(i).Tag.Get(helmFieldTag)
-		if helmValueKey != "" && valueOf.Field(i).Len() > 0 {
-			rawArgs = append(rawArgs, fmt.Sprintf("%s=%s", helmValueKey, valueOf.Field(i)))
+		if len(helmValueKey) > 0 && valueOf.Field(i).Len() > 0 {
+			if helmValueKey == "*" {
```
Comment (zyy17) [inline pkg/helm/helm.go:102]: todo: I think it's necessary to write the unit test for `*` case.

File: pkg/cmd/gtctl/cluster/create/create.go (hunk 1 above)
Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/create/create.go]: great idea, also should be pointed out in its usage

File: pkg/helm/helm.go (hunk 2 above)
Comment (shawnh2) [reply pkg/helm/helm.go:102]: got it

Comment (shawnh2) [reply pkg/helm/helm.go:102]: done

File: pkg/cmd/gtctl/cluster/create/config.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+		for _, value := range values {
+			value = strings.Trim(value, " ")
+			config := strings.SplitN(value, ".", 2)
+			configPrefix = config[0]
+			if len(config) == 2 {
+				configValue = config[1]
+			} else {
+				configValue = configPrefix
+			}
```
Comment (daviderli614) [inline pkg/cmd/gtctl/cluster/create/config.go:62]: input: `etcd.for=bar`

question: Why does not return failure when the length is not equal 2？

File: pkg/cmd/gtctl/cluster/create/config.go [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+			switch configPrefix {
+			case configOperator:
+				operatorConfig = append(operatorConfig, configValue)
+			case configCluster:
+				clusterConfig = append(clusterConfig, configValue)
+			case configEtcd:
+				etcdConfig = append(etcdConfig, configValue)
+			default:
+				clusterConfig = append(clusterConfig, value)
```
Comment (daviderli614) [inline pkg/cmd/gtctl/cluster/create/config.go:72]: thought: Why here is clusterConfig by default.

File: pkg/cmd/gtctl/cluster/create/config.go (hunk 3 above)
Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/create/config.go:62]: cause input `etcd.foo=bar`, the `foo=bar` is for etcd config.

if user input `foo=bar`, apparently it does not specific any config type whatever is going to set. so in that case, i wish gtctl will treat this case as it is for `cluster`, which is the default config type.

so by default, input like `foo=bar` is for `cluster`

File: pkg/cmd/gtctl/cluster/create/config.go (hunk 4 above)
Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/create/config.go:72]: ditto

File: pkg/cmd/gtctl/cluster/create/config.go (hunk 3 above)
Comment (shawnh2) [reply pkg/cmd/gtctl/cluster/create/config.go:62]: which is also going to clarify in its usage

**Review by zyy17 — APPROVED** [review]
LGTM

**Comment by zyy17** [comment]
@daviderli614 PTAL

**Pull request merged**